model/license_test.go

```diff
@@ -158,6 +159,7 @@ func TestLicenseToFromJson(t *testing.T) {
 			Company: NewId(),
 		},
 		Features: &f,
+		IsTrial:  true,
```

Comment:
This matches the new field added in https://github.com/mattermost/license-generator/pull/15.